pkg/manifest/manifest_test.go

```diff
@@ -109,7 +109,7 @@ func TestManifest(t *testing.T) {
 			} else {
 				// otherwise if we expect the manifest to be invalid/unrecognised then
 				// the error should match our expectations.
-				if !errors.As(err, &tc.expectedError) {
+				if !errors.Is(err, tc.expectedError) {
```

Comment:
I tried this change locally and see:

<img width="1068" alt="Screenshot 2022-08-03 at 09 14 13" src="https://user-images.githubusercontent.com/180050/182558759-5a259efa-5c95-4d04-b20f-016a037e6286.png">

Reply:
oh nevermind, just noticed that this is errors.Is not errors.As